examples/workflow-test/src/graph/elements/edge.po.ts

```diff
@@ -0,0 +1,9 @@
+import { Clickable, Mix, RoutingPointCapability } from '@eclipse-glsp/glsp-playwright-core/extension';
+import { EdgeMetadata, PEdge } from '@eclipse-glsp/glsp-playwright-core/glsp';
+
+const Base = Mix(PEdge).flow(Clickable).capability(RoutingPointCapability).build();
+
+@EdgeMetadata({
+    type: 'edge'
+})
+export class Edge extends Base {}
```

Comment:
For the purpose of the example: Do we want/need a weighted edge here as well? Because that was kind of skipped.

Reply:
Yes, we can add it maybe for further testing scenarios